modules/twinkleblock.js

```diff
@@ -1096,7 +1134,11 @@ Twinkle.block.blockGroupsPartial = [
 	{
 		label: 'Partial block reasons',
 		list: [
-			{ label: 'Partial block', value: 'uw-pblock', selected: true }
+			{ label: 'Generic partial block (custom reason)', value: 'uw-pblock', selected: true },
+			{ label: 'Generic partial block (custom reason) - indefinite', value: 'uw-pblockindef' },
+			{ label: 'Edit warring', value: 'uw-ewpblock', selected: true },
+			{ label: 'Misusing multiple aounts', value: 'uw-acpblock', selected: true },
```

Comment:
fix typo in label -> "misusing"

Reply:
The other one yes, but not this one?

Reply:
@Amorymeltzer "accounts" welp, seeing things it seems.

Reply:
Oy yes.  Common issue with me, involving a shortcut for er/contract-region.  Finally changed in my `.emacs`, thanks.